android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/constants/MyLocationTracking.java

```diff
@@ -29,25 +29,8 @@
     public static final int TRACKING_NONE = 0x00000000;
 
     /**
-     * Tracking the only the location of the user.
+     * Tracking user, {@link MapView} will reposition to center of {@link com.mapbox.mapboxsdk.views.UserLocationView}
```

Comment:
Hmm "Tracking the location of the user" would be a better start, and move the reposition bit to a second sentance